docs/types/entity_action_types/drop_inventory.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Drop Inventory (Entity Action Type)
+date: 2022-6-07
+---
+
+# Drop Inventory
+
+[Entity Action Type](../entity_action_types.md)
+
+Drops items directly from any spot in a player's inventory.
```

Comment:
```suggestion
Drops items from either the entity's inventory or a power that uses the [Inventory (Power Type)](../../power_types/inventory.md).
```
This entity action type is generalized to entities that can have inventories. It also works for powers that's using the `inventory` power type